apiserver/service/minio_server.go

```diff
@@ -730,6 +731,46 @@ func (m *minioAPI) ReadCSVLines(ctx *gin.Context) {
 	ctx.JSON(http.StatusOK, result)
 }
 
+// @Summary Get a download link
+// @Schemes
+// @Description Get a download link
+// @Tags MinioAPI
+// @Accept json
+// @Produce json
+// @Param namespace header string true  "Name of the bucket"
+// @Param bucketPath query string true "Path of the bucket"
+// @Param fileName query string true "Name of the file"
+// @Success 200 {object} map[string]string
+// @Failure 400 {object} map[string]string
+// @Failure 500 {object} map[string]string
+// @Router /model/files/downloadlink [get]
+func (m *minioAPI) GetDownloadLink(ctx *gin.Context) {
+	source, err := common.SystemDatasourceOSS(ctx.Request.Context(), nil, m.client)
+	if err != nil {
+		klog.Errorf("failed to get datasource %s", err)
+		ctx.AbortWithStatusJSON(http.StatusInternalServerError, gin.H{
+			"message": err.Error(),
+		})
+		return
+	}
+
+	bucket := ctx.GetHeader(namespaceHeader)
+	bucketPath := ctx.Query(bucketPathQuery)
+	fileName := ctx.Query("fileName")
+	objectName := fmt.Sprintf("%s/%s", bucketPath, fileName)
+
+	u, err := source.Core.PresignedGetObject(ctx.Request.Context(), bucket, objectName, time.Hour*24*7, url.Values{})
+	if err != nil {
+		klog.Errorf("failed to get generate download link %s", err)
```

Comment:
1. failed to get generate download link =》two verbs here, it should be 'generate'?
2. 24*7 hour is some too long, I think 12 hour is enough if we'll generate a new one for each API call.